src/logics/radiant-v2/tokens/data/arbitrum.json

```diff
@@ -1,102 +1,47 @@
 {
-  "WBTC": {
-    "chainId": 42161,
-    "address": "0x2f2a2543B76A4166549F7aaB2e75Bef0aefC5B0f",
-    "decimals": 8,
-    "symbol": "WBTC",
-    "name": "Wrapped BTC"
-  },
   "rWBTC": {
     "chainId": 42161,
     "address": "0x727354712BDFcd8596a3852Fd2065b3C34F4F770",
     "decimals": 8,
     "symbol": "rWBTC",
     "name": "Radiant interest bearing WBTC"
   },
```

Comment:
We can remove all complex assets in arbitrum.json except `rUSDC` and `rWETH` which are used for test.

Reply:
Keep another `rUSDCn` for the `src/logics/radiant-v2/service.test.ts` `Test toAsset`.
 I change to  `rUSDC -> USDC.e` to `rUSDCn -> USDC`, becasue `USDC.e` the on-chain data is different from our data

```
-{"chainId":42161,"address":"0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8","decimals":6,"symbol":"USDC","name":"USD Coin (Arb1)"}
+{"chainId":42161,"address":"0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8","decimals":6,"symbol":"USDC.e","name":"Bridged USDC"}
```